src/components/EmptyState/README.md

```diff
@@ -163,6 +163,29 @@ Use to explain a single feature before merchants have used it.
 </EmptyState>
 ```
 
+### Empty state with subdued footer
+
+Use to provide additional but non-critical context for a new product or feature. Can also be used to include a subdued CTA for secondary or tertiary actions.
+
+```jsx
+<EmptyState
+  heading="Manage your inventory transfers"
+  action={{content: 'Add transfer'}}
+  secondaryAction={{content: 'Learn more', url: 'https://help.shopify.com'}}
+  footerContent={
+    <p>
+      If you don’t want to add a transfer, you can import your inventory from{' '}
+      <Link monochrome url="/settings">
+        settings
```

Comment:
This is a complete sentence so it needs a period.

Reply:
updated 🏓